### DISCUSSION TITLE
Interface for unit testing

### DISCUSSION THREAD

**Discussion by otis-viles** [silentpartnersoftware/Keycloak.Net#58] [General]
I noticed there's an unmerged PR to add an interface to KeycloakClient to allow mocking it in unit tests. I was curious what it needed to be able to be merged?

**Comment by societ-campbellwray** [comment]
Hi @otis-viles, I'm not sure what PR you are referring to, do you have a link?

There is issue #16, but no work has been started on that yet.

**Comment by otis-viles** [comment]
/facepalm

I’m sorry, it was a branch and not a PR; I think I was so happy to see it that I crisscrossed the 2 things in my head.
